server/game/cards/01-DTR/ThePharmacy.js

```diff
@@ -0,0 +1,25 @@
+const DeedCard = require('../../deedcard.js');
+const GameActions = require('../../GameActions/index.js');
+
+class ThePharmacy extends DeedCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'The Pharmacy',
+            playType: ['noon'],
+            cost: ability.costs.bootSelf(),
+            target: {
+                activePromptTitle: 'Select dude to unboot',
+                cardCondition: { location: 'play area', booted: true },
+                cardType: ['dude']
+            },
+            handler: context => {
+                this.game.resolveGameAction(GameActions.unbootCard({ card: context.target }), context);
+                this.game.addMessage('{0} uses {1} to unboot {2}', context.player, this, context.target);
```

Comment:
This is working fine, but I would like to have it to follow standard I am setting in the doc for implementing cards. Many of cards already implemented by me will also be changed.
That is if the handler is simple with one message, it should go to the `message` property.
```javascript
target: {
  // ...
},
message: context => this.game.addMessage('{0} uses {1} to unboot {2}', context.player, this, context.target),
handler: context => {
   // ...
}
```